util/start_test

```diff
@@ -1424,7 +1443,6 @@ def preprocess_options():
             escape_next_argument(sys.argv, index)
         index += 1
 
-
 def escape_next_argument(args, index):
     next = args[index + 1].strip()
```

Comment:
``start_test --compopts`` raises an unhandled python ``list index out of range`` error here.

Reply:
That's not a new issue, but good catch -- I'll lump the fix in for that into this PR too

Reply:
fixed